contracts/ReaperAutoCompoundScreamLeverage.sol

```diff
@@ -230,36 +229,41 @@ contract ReaperAutoCompoundScreamLeverage is ReaperBaseStrategy {
         _onlyStrategistOrOwner();
         withdrawSlippageTolerance = _withdrawSlippageTolerance;
     }
+
+    /**
+     * @dev Sets the swap path to go from {WFTM} to {want}.
+     */
+    function setWftmToWantRoute(address[] calldata _newWftmToWantRoute) external {
+        _onlyStrategistOrOwner();
+        require(_newWftmToWantRoute[0] == WFTM, "bad route");
+        require(_newWftmToWantRoute[_newWftmToWantRoute.length - 1] == want, "bad route");
+        delete wftmToWantRoute;
+        wftmToWantRoute = _newWftmToWantRoute;
+    }
+
     /**
      * @dev Function to retire the strategy. Claims all rewards and withdraws
      *      all principal from external contracts, and sends everything back to
      *      the vault. Can only be called by strategist or owner.
      *
      * Note: this is not an emergency withdraw function. For that, see panic().
      */
-    function retireStrat() external {
+    function retireStrat() external doUpdateBalance {
         _onlyStrategistOrOwner();
         _claimRewards();
         _swapRewardsToWftm();
         _swapToWant();
 
-        uint256 maxAmount = type(uint256).max;
-        _deleverage(maxAmount);
-        _withdrawUnderlyingToVault(maxAmount, false);
-        updateBalance();
+        _deleverage(type(uint256).max);
+        _withdrawUnderlyingToVault(balanceOfPool, false);
```

Comment:
sending max uint to `withdrawUnderlyingToVault` overflows when doing the slippage check.